test cases/common/18 includedirxyz/do_not_delete

```diff
@@ -0,0 +1 @@
+This file is to ensure this directory exists
```

Comment:
File has cosmetic issues.

Reply:
What do you mean?

Reply:
If you view it in the github UI, it has a red circle thing.

In `git log -p`, it looks like this:

```diff
diff --git a/test cases/common/18 includedirxyz/do_not_delete b/test cases/common/18 includedirxyz/do_not_delete
new file mode 100644
index 0000000000..e0ebf40da8
--- /dev/null
+++ b/test cases/common/18 includedirxyz/do_not_delete  
@@ -0,0 +1 @@
+This file is to ensure this directory exists
\ No newline at end of file
```

It should have a newline char.